pages/pipelines/notifications.md.erb

```diff
@@ -99,7 +105,7 @@ To send a notification to one particular workspace and channel, you can specify
 
 ```yaml
 notify:
-  - slack: buildkite-community#general
+  - slack: "buildkite-community#general"
```

Comment:
These aren’t required to be quoted, but maybe standardising on quoted form will reduce errors?

Reply:
yup. quoting isn't essential in this case, but consistency was my thinking here